Pod/Classes/Acknowledgements/AcknowlegementsListViewModel.swift

```diff
@@ -73,7 +73,11 @@ private extension AcknowledgementsListViewModel {
         // First and last elements in the settings bundle are not needed (title and empty row).
         // these are broken up to speed up compile times
         let innerRange = 1..<(specifiers.count - 1)
+        #if swift(>=4.1)
+        let rawAcknowledgements = specifiers[innerRange].compactMap(AcknowledgementViewModel.init(dictionary:))
```

Comment:
#pp define a compatibility alias for `compactMap` on Swift 3.x, like we did in BonMot, so this line doesn't have to be conditionally compiled